### ISSUE TITLE
Changing folders in Windows Client

### ISSUE THREAD

**Issue by gjf** [keybase/keybase-issues#2865]
Windows Client installs itself in %profiledir%\AppData\Local\Keybase and sets a home folder for all stuff and "file system" as %profiledir%\AppData\Roaming\Keybase

It is really inconvenient: I am using SSD as a system disk and I don't want to have additional read-write operations because of Keybase Client.

Is it possible to add a possibility to chose installation and home folders in the client?

**Comment by zQueal** [comment]
Although I very highly doubt that Keybase would put enough read/write stress on an SSD to remarkably--if at all--lower the life of a drive, this would be a very cool feature.

**Comment by gjf** [comment]
Yes, but anyway - why I should use 10 Gb of SSD storage when I have HDD?

**Comment by cjb** [comment]
The 10GB of Keybase storage doesn't live on your disk, it lives in the cloud.

**Comment by gjf** [comment]
Oh yeah, today I have received an update - 113 Mb file! Thank you very much, but I don't want to dl and write 0,1 Gb every time on SSD during your testing period.
